BLAG.Common/Models/AnswerPicture.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BLAG.Common.Models
+{
+    class AnswerPicture : AnswerBase
+    {
+        public IList<Object> Picture;
+        public Object CorrectObject;
```

Comment:
Because a picture could be big, it would be better to store index of the picture in the IList, instead of the whole picture, I think.